app/views/tags/edit.html.erb

```diff
@@ -84,7 +84,7 @@
 
 <% if current_user.has_role?(:super_admin) || current_user.has_role?(:admin) %>
   <center>
-    <h1><a href="/admin/tags/<%= @tag.id %>" data-no-instant><%= @tag.name %> admin</a></h1>
+    <h1><a href="/admin/tags/<%= @tag.id %>/edit" data-no-instant><%= @tag.name %> admin</a></h1>
```

Comment:
**_Nit-pick_**: I feel like this link could be slightly more descriptive -- how do folks feel about updating this to read "<tag name> admin page" instead (or something more descriptive)?

**BEFORE PROPOSED CHANGE:**
![Screen Shot 2021-01-22 at 4 03 01 PM](https://user-images.githubusercontent.com/32834804/105558227-4e7f1580-5ccb-11eb-867c-13a4c3400747.png)

**AFTER PROPOSED CHANGE:**
![Screen Shot 2021-01-22 at 4 04 11 PM](https://user-images.githubusercontent.com/32834804/105558284-7a020000-5ccb-11eb-93af-e8e4d475bdc5.png)

Reply:
This is exactly the kind of nit-pick that I like -- agree with you completely that this isn't very clear. Either option is an improvement, but I like `[tag]'s admin page` the best as it seems the most clear to me! 🚀

Reply:
Awesome! I've updated the link to read `[tag]'s admin page`, @vaidehijoshi . 😄